Cargo.toml

```diff
@@ -46,20 +55,28 @@ tracing-subscriber = "0.3"
 
 
 # Stable FVM dependencies from crates.io
+fvm = { version = "3.4", default-features = false }     # no opencl feature or it fails on CI
 fvm_ipld_blockstore = "0.1"
 fvm_ipld_encoding = "0.3"
 fvm_ipld_car = "0.6"
 fvm_ipld_hamt = "0.6"
+fvm_shared = { version = "3.3", features = ["crypto"] }
+
 # Using 0.8 because of ref-fvm.
 # 0.9 would be better because of its updated quickcheck dependency.
 # 0.10 breaks some API.
 cid = { version = "0.8", features = ["serde-codec", "std"] }
 
-# The following are on crates.io but they might be pre-releases.
-fvm = { version = "3.1", default-features = false }     # no opencl feature or it fails on CI
-fvm_shared = { version = "3.1", features = ["crypto"] }
+# Depending on the release cycle, this dependency might want an earlier version of the FVM.
```

Comment:
I don't know if we will regret it in the long-run, but something we could do is to bring all the frc42 that we need to https://github.com/consensus-shipyard/fvm-utils, so we can upgrade accordingly. This is the approach we are following for actor runtimes. We also use extensively frc42 in actors, so maybe this helps. Instead of using the external dependency we can fork the relevant code that we need into our `fvm-utils`.

(Been there, and I hate this dependency dance, is a time drain)

Reply:
Hm, I didn't know you use it that much. It was released last month, so it doesn't seem to lag _that_ much. Would it be possible to jump in and do a release, could they just be busy? 

But yeah, if there is pain, we have that option too. I don't follow what features there are in the FVM releases, so I'm not sure we need to be on the bleeding edge. For now I think we can save ourselves the extra burden.